src/__tests__/datasource.test.ts

```diff
@@ -14,7 +14,7 @@ type QueryTargetLike = Partial<CogniteQuery>;
 
 const ds = getMockedDataSource();
 const { backendSrv, templateSrv } = ds;
-const { Asset, Custom, Timeseries } = Tab;
+const { Asset, Custom, Timeseries, ExtractionPipelines } = Tab;
```

Comment:
```suggestion
const { Asset, Custom, Timeseries } = Tab;
```
doesn't seem to be used?